lib/utils/middlewares/fiscalcode.ts

```diff
@@ -13,19 +16,11 @@ import { IResponseErrorValidation, ResponseErrorValidation } from "../response";
 export const FiscalCodeMiddleware: IRequestMiddleware<
   IResponseErrorValidation,
   FiscalCode
-> = request => {
-  const fiscalCode: string = request.params.fiscalcode;
-  if (FiscalCode.is(fiscalCode)) {
-    return Promise.resolve(
-      right<IResponseErrorValidation, FiscalCode>(fiscalCode)
-    );
-  } else {
-    const validationErrorResponse = ResponseErrorValidation(
-      "Bad request",
-      `The fiscal code [${fiscalCode}] is not valid.`
-    );
-    return Promise.resolve(
-      left<IResponseErrorValidation, FiscalCode>(validationErrorResponse)
+> = request =>
```

Comment:
can it be RequiredParamMiddleware("fiscalcode", FiscalCode); ?

Reply:
actually yes! good catch

Reply:
done